Makefile

```diff
@@ -23,55 +23,35 @@ OPERATOR_NAMESPACE=openshift-operators-redhat
 DEPLOYMENT_NAMESPACE=openshift-logging
 REPLICAS?=0
 
-# go source files, ignore vendor directory
-SRC = $(shell find . -type f -name '*.go' -not -path "./vendor/*")
+.PHONY: all build clean fmt generate run sec test-e2e test-unit
 
-#.PHONY: all build clean install uninstall fmt simplify check run
-.PHONY: all build clean fmt simplify run sec
+all: build
 
-all: build #check install
+GOSEC_RELEASE?=2.2.0
+bin/gosec:
+	curl -sSfL  https://github.com/securego/gosec/releases/download/v${GOSEC_RELEASE}/gosec_${GOSEC_RELEASE}_$(shell uname -s | tr '[:upper:]' '[:lower:]')_amd64.tar.gz | tar -xz -C ./bin
+	chmod +x $@
 
-operator-sdk: get-dep
-	@if ! type -p operator-sdk ; \
-	then if [ ! -d $(GOPATH)/src/github.com/operator-framework/operator-sdk ] ; \
-	  then git clone https://github.com/operator-framework/operator-sdk --branch master $(GOPATH)/src/github.com/operator-framework/operator-sdk ; \
-	  fi ; \
-	  cd $(GOPATH)/src/github.com/operator-framework/operator-sdk ; \
-	  make dep ; \
-	  make install || sudo make install || cd commands/operator-sdk && sudo go install ; \
+bin/imagebuilder:
+	@if [ $${USE_IMAGE_STREAM:-false} = false ] ; \
+	then GOFLAGS="" GO111MODULE=off go get -u github.com/openshift/imagebuilder/cmd/imagebuilder ; \
 	fi
 
-gendeepcopy: operator-sdk
-	@operator-sdk generate k8s
-
-imagebuilder:
-	@if [ $${USE_IMAGE_STREAM:-false} = false ] && ! type -p imagebuilder ; \
-	then go get -u github.com/openshift/imagebuilder/cmd/imagebuilder ; \
-	fi
-
-get-dep:
-	@if ! type -p dep ; then \
-		cd $(GOPATH) ; \
-		if [ ! -d $(GOPATH)/bin ]; then \
-			mkdir bin; \
-			fi; \
-			curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh ; \
-	fi
+OPERATOR_SDK_RELEASE?=v0.16.0
+bin/operator-sdk:
+	curl -sSfL -o $@ https://github.com/operator-framework/operator-sdk/releases/download/${OPERATOR_SDK_RELEASE}/operator-sdk-${OPERATOR_SDK_RELEASE}-$(shell uname -i)-linux-gnu
```

Comment:
I changed my approach slightly, check latest https://github.com/openshift/cluster-logging-operator/pull/429
Goal is to use the command from PATH if it's suitable rather than always force download/build, so I went back to dummy targets rather than file targets for commands, and use `$(MAKE) somecmd` in recepies that use the comand rather than having a make dependency (because dummy targets are always out of date and so always force their dependents to build)